test/contracts/root-element.spec.tsx

```diff
@@ -7,7 +7,8 @@ import {isReactComponent} from '../utils/is-react-component';
 const allComponents = Object.keys(WixReactComponents);
 const failingComponents = [
     'TreeView', 'NumberInput', 'Toggle', 'BirthdayPicker',
-    'RadioButton', 'RadioGroup', 'Portal', 'Popup', 'TimePicker'
+    'RadioButton', 'RadioGroup', 'Portal', 'Popup', 'TimePicker',
+    'Modal'
```

Comment:
Why add it to failing when it's a new component?

Reply:
Because it will never pass any of the tests